commands/Transfering/transfer.js

```diff
@@ -36,8 +36,31 @@ function successMessage(res, amount){
   broadcastOperation(amount, user, transferred_to);
 }
 
+function canRemoveByAngryPoints(value) {
+  if (value > 0) {
+    return false
+  }
+  var angryPointsLimit = Libs.ResourcesLib.userRes("angryPointsLimit")
+  if (-value > angryPointsLimit.value()) {
+    return false
+  }
+
+  var angryPoints = Libs.ResourcesLib.userRes("angryPoints")
+  if (angryPoints.value() + value < 0) {
+    // value will be already negative
+    return false
+  }
+  return true
+}
+
+function removeAngryPoints(amount) {
+  var angryPoints = Libs.ResourcesLib.userRes("angryPoints")
+  angryPoints.add(amount) //amount will be already negative
```

Comment:
I think it is must be:
angryPoints.add(Math.abs(removalPoints))

because we need positive value in angryPoints not negative